digid_eherkenning/management/commands/generate_digid_metadata.py

```diff
@@ -124,11 +129,16 @@ def add_arguments(self, parser):
             help="If True the metadata is printed to stdout. Defaults to False",
             default=False,
         )
+        parser.add_argument(
+            "--slo",
```

Comment:
We should add this option in the README example command :slightly_smiling_face:

Reply:
done :)